middleware_jwt_test.go

```diff
@@ -179,10 +179,10 @@ var jwtWithCentralDefNoPolicyBaseField = `
 
 `
 
-const JWTSECRET = "9879879878787878"
+const jwtSecret = "9879879878787878"
 
 // openssl genrsa -out app.rsa
-const JWTRSA_PRIVKEY = `
+const jwtRSAPrivKey = `
```

Comment:
Happy to have an internal commandment for this, even if it's just a prefix so that people reading the code can see it's a constant.